terminus-ui/form-field/src/form-field.component.md

```diff
@@ -6,20 +6,88 @@
 **Table of Contents**
 
 - [Basic usage](#basic-usage)
-    - [More...](#more)
+  - [control](#control)
+  - [floatLabel](#floatlabel)
+  - [hideRequiredMarker](#hiderequiredmarker)
+  - [hint](#hint)
+  - [noValidationOrHint](#novalidationorhint)
+  - [validateOnChange](#validateonchange)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 
 ## Basic usage
 
-Use the thing...
+`TsFormFieldComponent` is a component used to wrap several other components that could be used within a form and apply common styles. 
```

Comment:
> Use the thing...

This wasn't descriptive enough for you?!?! 😂 Glad to finally add some real docs! 💪